docs/content/preview/releases/techadvisories/ta-reol-24.md

```diff
@@ -0,0 +1,110 @@
+---
+title: TA-REOL-24
+headerTitle: Replicated End of Life
+headcontent: 30 Apr 2024
+type: docs
+showRightNav: true
+cascade:
+  unversioned: true
+menu:
+  preview_releases:
+    identifier: ta-reol-24
+    weight: 1
+rightNav:
+  hideH2: true
+type: docs
+---
+
+|          Product           |  Affected Versions   | Fixed In |
+| :------------------------- | :------------------ | :------- |
+| {{<product "yba">}}  | {{<release "all">}} | {{<release "2.18, 2.20+">}}      |
+
+## Critical Considerations
+
+A YugabyteDB Anywhere (YBA) Installer license must be acquired prior to attempting any migration. Please contact {{% support-platform %}} for a license request.
+
+## Description
+
+YugabyteDB Anywhere will end support for Replicated installation at the end of 2024.
+This means updates will no longer be available, preventing YBA and YugabyteDB database upgrades, in perpetuity.
+
+## Mitigation
+
+Following are detailed upgrade schedules you should reference when planning your migration.
+
+1. Upgrade YBA instances to the latest v2.18 or v2.20 using Replicated.
+1. Migrate from the Replicated-based install of YBA to a YBA-Installer-based install of YBA using the [Migration Steps](../../../yugabyte-platform/install-yugabyte-platform/install-software/installer/#migrate-from-replicated).
+
+## Details
+
+To guarantee interoperability, upgrading to a YBA version greater than or equal to latest v2.18 may require additional actions, such as upgrading the database cluster(s) version (also known as universes), or performing an upgrade to Docker.
+
+For more details, refer to [YBA version compatibility with YugabyteDB](../../../releases/yba-releases/#compatibility-with-yugabytedb).
+
+{{< warning title="Important" >}}
+Please do not upgrade your YBA to a version that no longer supports your currently running YugabyteDB database versions.
+{{< /warning >}}
+
+If there are no version compatibility issues, YBA upgrades should be performed before any additional upgrades to the database.
+
+When performing step 1 of the mitigation, review the [prerequisites](../../../yugabyte-platform/install-yugabyte-platform/prerequisites/default) for Replicated based installs to avoid any potential upgrade issues.
```

Comment:
```suggestion
When performing step 1 of the mitigation, review the [prerequisites](../../../yugabyte-platform/install-yugabyte-platform/prerequisites/default) for Replicated-based installs to avoid any potential upgrade issues.
```